pkg/component/observability/logging/vali/vali_test.go

```diff
@@ -1033,174 +965,66 @@ wait
 	return configMap
 }
 
-func getHVPA(isRBACProxyEnabled bool) *hvpav1alpha1.Hvpa {
-	controlledValues := vpaautoscalingv1.ContainerControlledValuesRequestsOnly
-	containerPolicyOff := vpaautoscalingv1.ContainerScalingModeOff
-
-	obj := &hvpav1alpha1.Hvpa{
+func getVPA(isRBACProxyEnabled bool) *vpaautoscalingv1.VerticalPodAutoscaler {
+	vpa := &vpaautoscalingv1.VerticalPodAutoscaler{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      valiName,
+			Name:      valiName + "-vpa",
 			Namespace: namespace,
-			Labels: utils.MergeStringMaps(getLabels(), map[string]string{
-				v1beta1constants.LabelObservabilityApplication: valiName,
-			}),
```

Comment:
Previously the HVPA had also the label `observability.gardener.cloud/app: vali`. I don't know whether there is a webhook on (H)VPA selecting by this label. Maybe not, but it might make sense to preserve the label. It was added with https://github.com/gardener/gardener/pull/9385.

cc @nickytd